projects/turing-yearbook.md

```diff
@@ -38,19 +40,25 @@ Notice that we have multiple stylesheets - one for each component. You are welco
 
 Create a Cohort component, which will act as a container that renders each of our Person cards. For now, the Cohort being displayed will be "Staff". Figure out how to pass information from `App.js` to `Cohort.js`!
 
+Then create a Person component. Each Person card will display: the image, the name, the quote, and the superlative (see the screenshot above for an idea).
+
+In your Cohort component, create Person cards that have the information each needs to display properly.
+
```

Comment:
Will you add a note that their tests should still pass or be updated to still pass, even with this new data?